plugins/obs-ffmpeg/data/locale/pl-PL.ini

```diff
@@ -35,6 +35,7 @@ ColorRange="Zakres kolorów YUV"
 ColorRange.Auto="Automatycznie"
 ColorRange.Partial="Częściowy"
 ColorRange.Full="Pełny"
+DisableSeeking="Wyłącz przewijanie"
```

Comment:
Please don't make non-English language file changes via GitHub.  Instead, [please use Crowdin](https://obsproject.com/forum/threads/how-to-contribute-translations-for-obs.16327/).